docs/getting-started.md

```diff
@@ -0,0 +1,179 @@
+      - `snowflake`
+      - `spark`
+
+## Configure an Operator
+
+After deciding [which Operator best fits your use case](#operator-use-cases), follow the Operator-specific instructions below to configure it.
+
+### Data Frame Operator
+
+1. Import the Operator.
+
+    ```python
+    from great_expectations_provider.operators.validate_dataframe import (
+        GXValidateDataFrameOperator,
+    )
+    ```
+
+2. Instantiate the Operator with required and optional parameters.
+
+    ```python
+    my_data_frame_validation = GXValidateDataFrameOperator(
+        task_id="my_data_frame_validation",
+        configure_dataframe=lambda: pd.read_csv(my_data_file),
+        expect=my_expectation_suite,
+    )
+    ```
+
+    - **`task_id`**: alphanumeric display name for the task in the Airflow UI.
+    - **`configure_dataframe`**: function that returns a DataFrame to pass data to the Operator.
+    - **`expect`**: either a [single Expectation](https://docs.greatexpectations.io/docs/core/define_expectations/create_an_expectation) or an [Expectation Suite](https://docs.greatexpectations.io/docs/core/define_expectations/organize_expectation_suites) to validate against your data. 
+    - **`result_format` (optional)**: dictionary that sets the [verbosity of returned Validation Results](https://docs.greatexpectations.io/docs/core/trigger_actions_based_on_results/choose_a_result_format/). Defaults to `SUMMARY`.
+    - **`context_type` (optional)**: accepts `ephemeral` or `cloud` to set the [Data Context](https://docs.greatexpectations.io/docs/core/set_up_a_gx_environment/create_a_data_context) used by the Operator. Defaults to `ephemeral`, which does not persist results between runs. To save and view Validation Results in GX Cloud, use `cloud` and complete the additional Cloud Data Context configuration below.
+
+    For more details, explore this [end-to-end code sample](https://github.com/klavavej/airflow-provider-great-expectations/tree/docs/great_expectations_provider/example_dags/example_great_expectations_dag.py#L134-L138).
+    
+3. If you use a Cloud Data Context, create a [free GX Cloud account](https://app.greatexpectations.io/) to get your [Cloud credentials](https://docs.greatexpectations.io/docs/cloud/connect/connect_python#get-your-user-access-token-and-organization-id) and then set the following Airflow variables.
+
+    - `GX_CLOUD_ACCESS_TOKEN`
+    - `GX_CLOUD_ORGANIZATION_ID`
+
+
+### Batch Operator
+
+1. Import the Operator.
+
+    ```python
+    from great_expectations_provider.operators.validate_batch import (
+        GXValidateBatchOperator,
+    )
+    ```
+
+2. Instantiate the Operator with required and optional parameters.
+
+    ```python
+    my_batch_validation = GXValidateBatchOperator(
+        task_id="my_batch_validation",
+        configure_batch_definition=my_batch_definition,
+        expect=my_expectation_suite,
+    )
+    ```
+
+    - **`task_id`**: alphanumeric display name for the task in the Airflow UI.
```

Comment:
likewise, it may be worth mentioning that this name will also be used by the operator to name GX resources. It's not really relevant for ephemeral users, since they will never see the GX components backing the validation, but it _is_ relevant for cloud users.